### ISSUE TITLE
Missing Refund Request in Payment Intents

### ISSUE THREAD

**Issue by kwarcu** [thephpleague/omnipay-stripe#144]
As far as I can see Refund is still missing in PaymentIntents. Am I correct to assume that as long as it won't be added there (https://stripe.com/docs/payments/cards/refunds) the standard `\Omnipay\Stripe\Message\RefundRequest` will stop working after 14 September because it uses the deprecated `/charges` endpoint?

**Comment by zenichanin** [comment]
Yep I'm facing the same issue. With payment intents, you have to create a new refund using the following endpoint:
https://stripe.com/docs/api/refunds/create

That one accepts both `charge_id` and `payment_intent_id` so technically the `RefundRequest` could be altered to accept either one.
